docs/3.0rc/manage/cloud/manage-users/secure-access-by-ip-address.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Secure access by IP address
+description: Manage network access to Prefect Cloud accounts with IP Allowlists.
+---
+
+Prefect Cloud's [Enterprise plan](https://www.prefect.io/pricing) offers IP allowlisting to restrict access to
+Prefect Cloud APIs and UI at the network level by giving users the ability to manage an explicit list of allowed IP addresses for an account.
+
+To get started, use the Prefect CLI to add an IP address to the allowlist:
+
+<Note>To help prevent accidental account lockouts, updates to an allowlist requires the requestor's current IP address to be on the list.</Note>
+
+```bash
+prefect cloud ip-allowlist add <your-ip-address> --description "My home IP address"
+```
+
+The allowlist has a limit to how many entries can be added; 
```

Comment:
i think the limit should be included here

Reply:
I omitted it thinking it might be nice in-case it changes in the future. the CLI should surface the exact limit from the server once you try to pass it...

... 😅 whoops looks like a small change i made is swallowing that now - PR to fix that https://github.com/PrefectHQ/prefect/pull/15137

and yeah I'll go ahead and add the limit in here 👍